byway/src/main/webapp/routepage.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+if (document.readyState === 'loading') {  // Loading hasn't finished yet
+  let recs = getRecs();
+  document.addEventListener('DOMContentLoaded', recs);
```

Comment:
Do you understand the difference between these two lines:

```js
// 1
let recs = getRecs();
// 2
let recs = getRecs;
```

Reply:
1 assigns and calls the function, getRecs() whereas 2 just assigns the variable recs to the function without calling it.

Reply:
Ok, so what is being scheduled for the event listener in this code?